code/src/components/ThoughtList.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { formatDistance } from 'date-fns'
+
+export const ThoughtList=({thoughts, onLikesIncrease})=>{
```

Comment:
You could add some spacing here to make the code more clean/consistent. Just a suggestions - and really not the most important thing. Your code looks great!